tools/python/mwm_feature_compare.py

```diff
@@ -0,0 +1,56 @@
+from __future__ import print_function
+
+import argparse
+import os
+import subprocess
+import multiprocessing
+
+OMIM_ROOT = os.path.join(os.path.dirname(os.path.abspath(__file__)), '..', '..')
+FIND_FEATURE = os.path.join(OMIM_ROOT, 'tools', 'python', 'mwm', 'find_feature.py')
+
+
+def count_feature(mwm_path, feature_name):
+    p = subprocess.Popen(['/usr/bin/python', FIND_FEATURE, mwm_path, 'et', feature_name], stdout=subprocess.PIPE)
+    features_count = sum(1 for _ in p.stdout)
+    return features_count
+
+
+def compare_feature_num(args_tuple):
+    old_mwm, new_mwm, feature_name, threshold = args_tuple
+    old_feature_count = count_feature(old_mwm, feature_name)
+    new_feature_count = count_feature(new_mwm, feature_name)
+
+    if new_feature_count - old_feature_count < 0:
+         p_change = float(old_feature_count - new_feature_count) / old_feature_count * 100
```

Comment:
What if old is zero?

Reply:
count_feature returning lines count from stdout, thus:
```
old_feature_count ≥ 0
new_feature_count ≥ 0
```
If old_feature_count is equal 0, then min(new_feature_count - old_feature_count) = 0, and line with p_change is unreachable.

Reply:
I'd check the zero case anyway. This frees your from relying on this complex argument and will work even if the behaviour of surrounding code changes. But up to you.